adafruit_mma8451.py

```diff
@@ -126,9 +126,8 @@ def _read_into(self, address, buf, count=None):
         if count is None:
             count = len(buf)
         with self._device as i2c:
-            buf[0] = address & 0xFF
-            i2c.write(buf, end=1, stop=False)
-            i2c.readinto(buf, end=count)
+            i2c.write_then_readinto(bytes([address & 0xFF]), buf,
```

Comment:
`out_end=1` should be specified, I think.

Reply:
we dont have to because i dont use 'buf' anymore, i just create and pass in a single-byte array. oddly when i used buf, it wasnt working. i just did this instead of in-depth investigation on why i couldnt use buf for both output and input

Reply:
got it, i didn't read carefully enough

Reply:
np its confusing, ive mixed both because you write first, then read. anyways...yay!